### ISSUE TITLE
Make changelog trackable

### ISSUE THREAD

**Issue by norbert-csibra-msm** [kesava-wow/kuinameplates2#62]
Hello,

Just a quick suggestion, make the changelog trackabale with release numbers, because currently hard to figure out what changed in the different releases, especially between the beta releases.
We can check the changes on github with compare, but that's kind of silly.
So instead of this:

```
* add option to enable nameonly mode on all enemies
* add option to enable nameonly mode on the target
* add target/threat glow to nameonly mode
* add option to fade out friendly NPCs by default
* add option to fade out neutral "enemies" by default
* add option to fade out non-tracked nameplates by default
* add option to change size of the (bar) frame glow
* fix combat toggle options don't apply on login
* mouseover highlight no longer shows on target
```

Something like this:
```
2-9-beta-3
* add option to fade out friendly NPCs by default
* add option to fade out neutral "enemies" by default

2-9-beta-2
* mouseover highlight no longer shows on target

2-9-beta-1
* add option to enable nameonly mode on all enemies
* add option to enable nameonly mode on the target
* add target/threat glow to nameonly mode
* fix combat toggle options don't apply on login
```

**Comment by kesava-wow** [comment]
full historical changelogs can be found here: https://wow.curseforge.com/addons/kuinameplates/files/

**Comment by ncsibra** [comment]
The whole point was to not give a full historical log as changelog, but be able to see the changes per release.
E.g. you added a feature in a beta release which I wanted immediately, so I changed the release type in curse client to beta and downloaded the release, next time when you released a new beta, I checked the changelog and saw the same things that was already in the previous release and it was hard to figure out what is new in that release, so harder to decide if I want to download it or wait for the stable release.

A real changelog should looks like I wrote, just check some other addons, e.g.:
https://mods.curse.com/addons/wow/tradeskill-master#t1:changes
